{RepositoryName}/src/{Namespace}.Data/Repositories/MyVC3Module4DbContext.cs

```diff
@@ -0,0 +1,25 @@
+using EntityFrameworkCore.Triggers;
```

Comment:
is the filename "MyVC3Module4DbContext.cs" correct?

Reply:
Fixed